Formula/cava.rb

```diff
@@ -0,0 +1,40 @@
+class Cava < Formula
+  desc "Console-based Audio Visualizer for ALSA"
+  homepage "https://github.com/karlstav/cava"
+  url "https://github.com/karlstav/cava/archive/0.7.4.tar.gz"
+  sha256 "fefd3cc04d41b03ca416630cafadbfda6c75e2ca0869da1f03963dcb13e1ecb7"
+  license "MIT"
+  head "https://github.com/karlstav/cava.git", branch: "master"
+
+  depends_on "automake"
+  depends_on "fftw"
+  depends_on "iniparser"
+  depends_on "libtool"
+  depends_on "portaudio"
+
+  uses_from_macos "ncurses"
+
+  def install
+    # force autogen.sh to look for and use our glibtoolize
+    inreplace "autogen.sh", "libtoolize", "glibtoolize"
+
+    system "./autogen.sh"
+    ENV.append "CFLAGS", "-g -O0"
```

Comment:
Why is this needed? This won't make it past the `configure` script. The shims strip these flags after that.